cpu/cc26xx_cc13xx/power_arch.c

```diff
@@ -96,18 +96,24 @@ void power_enable_domain(const power_domain_t domain)
 void power_clock_enable_gpio(void)
 {
     PRCM->GPIOCLKGR = GPIOCLKGR_CLK_EN;
+    PRCM->GPIOCLKGS = GPIOCLKGS_CLK_EN;
+    PRCM->GPIOCLKGDS = GPIOCLKGDS_CLK_EN;
 
     prcm_commit();
 }
 
 void power_clock_enable_gpt(uint32_t tim)
 {
     PRCM->GPTCLKGR |= (1 << tim);
+    PRCM->GPTCLKGS |= (1 << tim);
+    PRCM->GPTCLKGDS |= (1 << tim);
 
     prcm_commit();
 }
 void power_clock_enable_i2c(void) {
     PRCM->I2CCLKGR = I2CCLKGR_CLK_EN;
+    PRCM->I2CCLKGS = I2CCLKGS_CLK_EN;
+    PRCM->I2CCLKGDS = I2CCLKGDS_CLK_EN;
```

Comment:
Do you really want to keep I2C powered on during sleep?

Reply:
Also, no `|=` here?

Reply:
Fixed at https://github.com/RIOT-OS/RIOT/pull/15134/commits/5d9f06cdd854c34fcd26779b1f654c519a474ef1 and https://github.com/RIOT-OS/RIOT/pull/15134/commits/276df2ed6a23e068c58f56bbd441044b6c381b88 respectively.

Reply:
Didn’t look into the data sheet yet, if this is a single purpose register we could save us the read-modify-write.

Reply:
The register actually has other fields so it makes sense to `|=`. However this could be a good place to apply bit-banding which I haven't tried yet.

Reply:
When you use the functions from [`bit.h`](https://github.com/RIOT-OS/RIOT/blob/master/cpu/cortexm_common/include/bit.h#L75), bit-banding will be used automatically.

But this is for another PR :wink: